docs/components/sections/Navbar.tsx

```
@@ -0,0 +1,389 @@
+                        >
+                            Docs
+                        </span>
+                    </a>
+
+                    {/* Desktop Navigation */}
+                    <nav className={"hidden lg:flex items-center"}>
+                        <div
+                            className={"relative text-sm"}
+                            onMouseEnter={handleProjectDocsEnter}
+                            onMouseLeave={handleProjectDocsLeave}
+                        >
```

Biome (2.4.4)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)